webpack/webpack.common.js

```diff
@@ -68,12 +68,7 @@ module.exports = async options => {
           },
           {
             test: /\.(jpe?g|png|gif|svg|woff2?|otf|ttf|eot|ppt|pdf|zip)$/i,
-            loader: 'file-loader',
-            options: {
-              digest: 'hex',
-              hash: 'sha512',
-              name: 'content/[name].[ext]',
-            },
+            type: 'asset/resource',
```

Comment:
This change resolved the oncokb-style icons not loading. I originally copied over the webpack config from oncokb-public, but that project uses Webpack v4, while we use Webpack v5.
Reference: https://webpack.js.org/guides/asset-modules/